app/scrapers/baden_wuerttemberg_landtag_scraper.rb

```diff
@@ -0,0 +1,223 @@
+require 'date'
+
+# FIXME
+
+module BadenWuerttembergLandtagScraper
+  BASE_URL = 'http://www.landtag-bw.de'
+  DETAIL_URL = 'http://www.statistik-bw.de/OPAL'
+
+  def self.extract_result_divs(page)
+    list = page.search('//div[@id="result"]//ol')
+    list.css('.result')
+  end
+
+  def self.extract_full_reference(div)
+    div.at_css('p').text.gsub(/\s+/, "").gsub(/g/, "").match(/(.+)-.+Datum/)[1].gsub(/\p{Z}+/, ' ').strip
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.